.github/workflows/update-s3-dependencies.yml

```diff
@@ -0,0 +1,78 @@
+name: Update S3 HTML dependencies for download.pytorch.org nightly and test
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - s3_management/update_dependencies.py
+  workflow_dispatch:
+    inputs:
+      dryrun:
+        description: 'Enable dry run mode'
+        required: true
+        type: choice
+        default: enabled
+        options:
+          - enabled
+          - disabled
+      package:
+        description: 'Package to update'
+        required: true
+        type: choice
+        default: torch
+        options:
+          - all
+          - torch
+          - triton
+          - torchtune
+          - torch_xpu
```

Comment:
Could I have vLLM here too? The package is in the list, they are for nightly vLLM build :)

Reply:
Sure, lets add vllm as well